components/player/VideoPlayer.tsx

```diff
@@ -1,15 +1,57 @@
-import React from 'react';
-import { View } from 'react-native';
+/* eslint-disable react/no-unstable-nested-components */
+import { useTheme } from '@react-navigation/native';
+import { ResizeMode, Video } from 'expo-av';
+import { Image } from 'expo-image';
+import React, { useRef } from 'react';
+import { Dimensions, StyleSheet, View } from 'react-native';
+import { VIDEO_URL } from '../../config/constants';
 
-// TODO: Add VideoPlayer component
-/**
- * -  Implement a video player card component
- * - Hint: use `expo-av` to implement the video player with hidden controls
- * - consider adding fullscreen player
- */
+const { width } = Dimensions.get('window');
 
 const VideoPlayer = () => {
-  return <View />;
+  const { colors } = useTheme();
+
+  const video = useRef(null);
+
+  return (
+    <View style={[styles.container, { backgroundColor: colors.card }]} testID="video-player">
+      <Video
+        testID="video-component"
+        ref={video}
+        style={styles.video}
+        source={{
+          uri: VIDEO_URL,
+        }}
+        useNativeControls
+        resizeMode={ResizeMode.CONTAIN}
+        isLooping
+        isMuted
```

Comment:
The volume control does not show up on my Android 12 phone. I cannot unmute the video.

![telegram-cloud-photo-size-4-5834497481893855095-y](https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/52ea58a2-b21b-494d-b286-24fed7f28fb3)

Reply:
I've changed from defaulting to `isMuted` and added an initial volume of `0.3`. this should now work with default device volume control buttons